onnxruntime/core/providers/nnapi/nnapi_builtin/builders/helper.cc

```diff
@@ -267,6 +268,39 @@ void GetFlattenOutputShape(const NodeUnit& node_unit, const Shape& input_shape,
   dim_2 = std::accumulate(input_shape.cbegin() + axis, input_shape.cend(), 1, std::multiplies<int32_t>());
 }
 
+// Get the bias size (C) of Gemm op
+// ANEURALNETWORKS_FULLY_CONNECTED only supports 1d bias
+// Will test if C of Gemm can be squeezed and return the 1d vector size after squeeze
+bool GetBiasSize(const Shape& c_shape, int32_t android_feature_level, uint32_t& size) {
```

Comment:
nit: GetBiasSize -> GetGemmBiasSize.  

Function name should clarify it's for Gemm now that it's located in a general purpose helpers class.

Reply:
Although if it's specific to Gemm does it need to be here or could it be in gemm_op_builder.cc?

Reply:
moved to gemm_op_builder.cc as a local helper.

also did similar updates for other helper methods only used inside individual op_builder, such as flatten.